integration_tests/loopback2/loopback2.go

```diff
@@ -52,6 +55,36 @@ func makeTap(name string, bandwidth int) l2.FrameReadWriteCloser {
 	return tap
 }
 
+// Waits for a set of devices to go offline.
+// Args:
+//  devices: The devices that we want to wait for.
+func waitForOffline(devices map[string]l2.FrameReadWriteCloser) {
+	// Spin until the devices actually appear.
+	for {
```

Comment:
Just do
```
for range time.Tick(10*millisecond)
```
and create a time.after and default select off of it with a timeout so it can fail?